examples/todo/data/database.js

```diff
@@ -39,8 +39,12 @@ export function getTodo(id) {
   return todosById[id];
 }
 
-export function getTodos() {
-  return todoIdsByUser[VIEWER_ID].map((id) => todosById[id]);
+export function getTodos(status = 'any') {
+  var todos = todoIdsByUser[VIEWER_ID].map((id) => todosById[id]);
```

Comment:
For consistency, can you drop the parens around `id` here (like below, where you're using an arrow function without parens around the param)?